{"uid":"8a5a0875afbb25d4","name":"test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","fullName":"tests.catalog.options.test_HU11_ES_POST_ProductOption#test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","historyId":"09c4f63e9a9984c5936c41cb895fd765","time":{"start":1755546992466,"stop":1755546993193,"duration":727},"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTU1NDY5OTAsImV4cCI6MTc1NTU1MDU5MCwicm9sZX...JwIUvtzMzg0D5uoU64-yjQOb5p7opGqvBvfxEXNO7C7UWQLQ9-ORNLLNgCQu6dMlxehpf8jiZ-0QM4JYc63wMcHCGz3BlGz9XYWPUGVeSs-H_vbE'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.medium\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:184: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [500]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1755546990075,"stop":1755546990076,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"auth_headers","time":{"start":1755546990076,"stop":1755546990889,"duration":813},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_add_options","time":{"start":1755546992465,"stop":1755546992465,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTU1NDY5OTAsImV4cCI6MTc1NTU1MDU5MCwicm9sZX...JwIUvtzMzg0D5uoU64-yjQOb5p7opGqvBvfxEXNO7C7UWQLQ9-ORNLLNgCQu6dMlxehpf8jiZ-0QM4JYc63wMcHCGz3BlGz9XYWPUGVeSs-H_vbE'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.medium\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:184: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [500]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","steps":[],"attachments":[{"uid":"f8be3bc479d42335","name":"log","source":"f8be3bc479d42335.txt","type":"text/plain","size":1970}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"setup_add_options::0","time":{"start":1755546993223,"stop":1755546993223,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"medium"},{"name":"tag","value":"boundary"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.catalog.options"},{"name":"suite","value":"test_HU11_ES_POST_ProductOption"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"2412-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.catalog.options.test_HU11_ES_POST_ProductOption"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":3,"passed":0,"unknown":0,"total":3},"items":[{"uid":"3efd1e3643b2a76c","reportUrl":"https://Elv500.github.io/Softsign/70//#testresult/3efd1e3643b2a76c","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755546839510,"stop":1755546839874,"duration":364}},{"uid":"8db98430c6c6fa81","reportUrl":"https://Elv500.github.io/Softsign/57//#testresult/8db98430c6c6fa81","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755544227625,"stop":1755544228436,"duration":811}}]},"tags":["boundary","functional","medium"]},"source":"8a5a0875afbb25d4.json","parameterValues":[]}